### DISCUSSION TITLE
스터디 - 16주차 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#125] [General]
🏥 스터디 하실 분은 댓글 또는 아래의 내용을 수정해주세요.

## 시간표
2022.04.07(목) - 오후 9시
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접,1:1, 1:다로 모의면접 진행(상대방이 공부한 것을 기준으로 2개 이상 질문 진행)
21:35 ~ 21:50 모의 면접 진행이 끝난 후, 토론 장소에 모여서 다른 분들에게 궁금했던 걸 토론을 진행
21:50 ~ 22:00 프론트엔드 모의 면접 질문 정리에 작성 후, 돌아가면서 소감 한마디 하고 스터디 종료 
  
## 남은 
- React - 버전 18 요약

## 주현
- React - React.PureComponent

## 영생
- Javascript - 일급함수

## 송현
- Git - flow
- CS - Process, Thread

## 수민
- React - Test

**Comment by userJu** [comment]
주현

React - React.PureComponent

**Comment by NAMU1105** [comment]
남은
React - 버전 18 요약

**Comment by whdudtod1273** [comment]
영생
Javascript - 일급함수

**Comment by ttumzzi** [comment]
수민
React - Test
